cloud-operator/templates/clusterrole.yaml

```
@@ -8,65 +8,20 @@ metadata:
     {{- include "cloud-operator.labels" . | nindent 4 }}
 rules:
 - apiGroups: [""]
-  resources: ["bindings", "componentstatuses", "configmaps", "endpoints", "events", "limitranges", "namespaces", "nodes", "persistentvolumeclaims", "persistentvolumes", "pods", "podtemplates", "replicationcontrollers", "resourcequotas", "serviceaccounts", "services"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["admissionregistration.k8s.io"]
-  resources: ["mutatingwebhookconfigurations", "validatingadmissionpolicies", "validatingadmissionpolicybindings", "validatingwebhookconfigurations"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["apiregistration.k8s.io"]
-  resources: ["apiservices"]
+  resources: ["configmaps", "endpoints", "namespaces", "nodes", "pods", "replicationcontrollers", "serviceaccounts", "services"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["apps"]
-  resources: ["controllerrevisions", "daemonsets", "deployments", "replicasets", "statefulsets"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["authentication.k8s.io"]
-  resources: ["tokenreviews"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["authorization.k8s.io"]
-  resources: ["localsubjectaccessreviews", "selfsubjectaccessreviews", "selfsubjectrulesreviews", "subjectaccessreviews"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["autoscaling"]
-  resources: ["horizontalpodautoscalers"]
+  resources: ["replicasets","deployments","statefulsets", "daemonsets"]
```

Super-nit: consistent spacing.
```suggestion
  resources: ["replicasets", "deployments", "statefulsets", "daemonsets"]
```